cernopendata/templates/cernopendata_records_ui/records/record_detail.html

```diff
@@ -192,6 +192,10 @@ <h2>System details</h2>
                 <label>Recommended release for analysis:</label>
                 {% endif %}
                 <span>{{record.system_details.release}}</span>
+                {% for container_image in record.system_details.container_images %}
+                    <p>{{ container_image.registry.capitalize() }} container image FQIN: {{ container_image.name }}</p>
```

Comment:
The current presentation looks like:

![cms-system-details](https://github.com/cernopendata/opendata.cern.ch/assets/517546/d7b1c8f1-8d90-4f51-8bec-9e2c9986254c)

I would rather write something like:

Recommended container images for analyses: (see _guide_):
- docker: `docker.io/cmsopendata/cmssw_4_2_8-slc5_amd64_gcc434:latest`
- gitlab: `gitlab-registry.cern.ch/cms-cloud/cmssw-docker-opendata/cmssw_4_2_8-slc5_amd64_gcc434:latest`

That could also contain the link to the guide in a rather compact manner